tests/Arr/MergeTest.php

```diff
@@ -8,6 +8,9 @@
 
 /**
  * @covers \EoneoPay\Utils\Arr::merge
+ *
+ * @uses \EoneoPay\Utils\Arr::set
```

Comment:
So, just so I'm clear here - under the proposed rules, if `@use` is _not_ set, then `\EoneoPay\Utils\Arr::set()` in this case would be counted in the code coverage?

Reply:
No, with the presence of `@covers` you are telling the coverage engine that code coverage should only be counted towards what you've specified.

When running phpunit with `--strict-coverage`, it will error out if there is code used that is not annotated as covered (in this case, `Arr::set`), and marks the test as risky unless you either opt in to code coverage with `@covers` or in this case, opt out of coverage explicitly with `@uses`

Reply:
I'm actually ok with this then, since using a class or method to set up the pre-conditions for the test is bad practice anyway  (my opinion is here: https://loyaltycorp.atlassian.net/wiki/spaces/COM/pages/676462656/Arrange+Act+Assert under Arrange / Anti-patterns)